### ISSUE TITLE
Is there a Compatibility list doc for kubeflow and k8s?

### ISSUE THREAD

**Issue by zhucan** [kubeflow/community#747]
like this:
![image](https://github.com/user-attachments/assets/fb2ba635-b17c-4da4-922a-b5e71b9af491)

**Comment by brunoR1000** [comment]
This might help you
https://bit.ly/3yXXzbX
Passcode: d5kp14h
You must have gcc compiler installed

**Comment by zhucan** [comment]
https://www.kubeflow.org/docs/releases/ this?

**Comment by juliusvonkohout** [comment]
Yes, check the kubeflow releases. For example https://github.com/kubeflow/manifests/releases/tag/v1.10.0

/close
